### DISCUSSION TITLE
Issues building an ISO locally

### DISCUSSION THREAD

**Discussion by sdwilsh** [kairos-io/kairos#875] [Q&A] [answered]
I'm having some issues trying to build the ISO so I can test changes locally for `ubuntu-20-lts`.  The build command I'm using is `./earthly.sh +iso --FLAVOR=ubuntu-20-lts`, but it fails trying to build the docs: https://ci.earthly.dev/logs?logId=65ab2a2b-5ebd-406f-9783-70b45927f93f

Am I using the right command to build this locally?  It seems like I shouldn't have a dependency problem because the project uses Earthly, but that kinda looks like what I'm hitting, alas.

**Comment by mauromorales** [comment]
I see it's somehow failing at fetching the submodule commit, which should be publicly available

```
fatal: Fetched in submodule path 'themes/docsy', but it did not contain 5597d435dc74ce68240e0c3871addf24567493b0. Direct fetching of that commit failed.
```

**Comment by sdwilsh** [comment, marked as answer]
I took a week off for a vacation, and now that I'm back, the latest code builds just fine 🤷

**Reply by mauromorales** [reply]
I know the feeling ^__^

**Reply by Itxaka** [reply]
I have seen that exact same issue from time to time, still have no idea what it means :D
